### ISSUE TITLE
Compatibility with package: yireo/magento2-webp2

### ISSUE THREAD

**Issue by jellesiderius** [yireo/Yireo_Webp2ForAmastyShopby#1]
Hi,

When trying to update the webp2 module, it shows that this plugin is not compatible with the webp2 module.

`    - yireo/magento2-webp2-for-amasty-shopby dev-master requires yireo/magento2-webp2 ^0.7 -> satisfiable by yireo/magento2-webp2[0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.7.5, 0.7.6, 0.7.7] but these conflict with your requirements or minimum-stability.`

**Comment by jissereitsma** [comment]
That's correct. This Amasty patch only worked with an older version of WebP, while newer versions of WebP work in combination with a NextGenImages extension. Could you confirm why you need the Amasty patch?

**Comment by malou72** [comment]
Hi,

First of all thank you for this great work which will bring performance to the loading times on sites.

My Magento 2 module work with a newer module Amasty_ShopBy version "2.15.3". 

On first page displaying, I see a loading because of the "swatch-option-loading" class that it don't remove on <picture> tag. 
Also when I'm working with filters pictures are displayed but not converted to <picture> tag.

All gd PHP libraries are enabled.
Is it the issue the module fixed before or do you have an idea ?

Thanks for your reply,

**Comment by jissereitsma** [comment]
@malou72 Are you reporting on the same issue as @jellesiderius which is involving a composer error? If so, could you please respond to that with the composer error details?

If you are reporting another issue, opening up another **Issue** here in this repository is a must. Otherwise, it becomes a mess. When opening up a new issue, could you add some further comments there? For instance, you mention *my Magento 2 module* - what is the name of that module? And how is it supposed to be working together with the Amasty module? If it is a typo, and you meant to say "Magento 2 project" instead, please make sure to remove all typos first. A troubleshooting tip would be to disable the Yireo module and confirm that everything is working (and check silly things like Developer Mode, caching, Full Page Cache disabled, etc - you know the basic stuff). Next, enable the Yireo module and disable the Amasty module. Are things working then? Next, enable both and mention precisely what happens. I hope this helps you. Of course, if you are **not** reporting a new issue, but only responding to the composer error, you can just respond to that issue here instead.
